pkgs/games/doom-ports/doomrunner/default.nix

```diff
@@ -0,0 +1,62 @@
+{ lib
+, stdenv
+, qtbase
+, qmake
+, makeDesktopItem
+, wrapQtAppsHook
+, fetchFromGitHub
+, fetchurl
+}:
+let
+  desktopItem = makeDesktopItem {
+    name = "DoomRunner";
+    desktopName = "DoomRunner";
+    comment = "Preset-oriented graphical launcher of ZDoom and derivatives";
+    categories = [ "Game" ];
+    icon = "DoomRunner";
+    type = "Application";
+    exec = "DoomRunner";
+  };
```

Comment:
I think this is not necessary to put this part in `let/in` and it can be moved inside `stdenv.mkDerivation`.